File: LeetCode/C#/1480. Running Sum of 1d Array.cs
public class Solution {
    public int[] RunningSum(int[] nums) {
        var runningSum = new int[nums.Length];
        
        for(var i = 0; i < nums.Length; i++){
            if(i == 0){
                runningSum[i] = nums[0];
            }
            else{
                runningSum[i] = runningSum[i-1] + nums[i];
            }
        }
        
        return runningSum;
    }
}

File: LeetCode/C++/1480. Running Sum of 1d Array.cpp
class Solution
{
public:
    vector<int> runningSum(vector<int> &nums)
    {
        int n = nums.size();
        vector<int> runningSum(n);
        runningSum[0] = nums[0];

        for (int i = 1; i < n; i++)
        {
            runningSum[i] = runningSum[i - 1] + nums[i];
        }

        return runningSum;
    }
};